lua/gitlab/init.lua

```diff
@@ -47,7 +47,8 @@ return {
   toggle_discussions = async.sequence({ info }, discussions.toggle),
   edit_comment = async.sequence({ info }, discussions.edit_comment),
   delete_comment = async.sequence({ info }, discussions.delete_comment),
-  toggle_resolved = async.sequence({ info }, discussions.toggle_resolved),
+  toggle_comment_resolved = async.sequence({ info }, discussions.toggle_comment_resolved),
```

Comment:
Agreed with your comment! Can you please remove this endpoint, use the "p" keybinding to resolve the discussion instead of "P" and then remove the `toggle_comment_resolved` keybinding and logic

Reply:
You can also simplify the PATCH `/comment` endpoint a little bit, by removing some of the other stuff:

```lua
	/* The PATCH can either be to the resolved status of
	the discussion or or the text of the comment */
	msg := "edit comment"
	if editCommentRequest.Comment == "" {
		options.Resolved = &editCommentRequest.Resolved
		msg = "update discussion status"
	} else {
		options.Body = gitlab.String(editCommentRequest.Comment)
	}
```

Reply:
I removed `toggle_comment_resolved` and updated the `PATCH` comment method. Should I keep the  option named `toggle_discussion_resolved` or you prefer `toggle_resolved` ?

Reply:
We want peoples' keybindings (if they picked something other than "p") to automatically work, so let's keep the name as `toggle_resolved` instead. You can test it by adding a `settings.discussion_tree.toggle_resolved` keybinding.

Reply:
Updated :ok_hand: .